presto-parser/src/main/java/com/facebook/presto/sql/tree/CurrentTime.java

```diff
@@ -24,10 +24,10 @@
 public class CurrentTime
         extends Expression
 {
-    private final Type type;
+    private final Operation operation;
     private final Integer precision;
 
-    public enum Type
+    public enum Operation
```

Comment:
"Function" ?

Reply:
Sure. It doesn't really matter much. Once we finish splitting the AST from IR, I plan to make each a different type of AST node and this will go away.